webapp/javascript/redux/reducers/continuous.ts

```diff
@@ -1,26 +1,69 @@
-import { Profile } from '@pyroscope/models/src';
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
-import { AppNames } from '@webapp/models/appNames';
-import { Query, brandQuery, queryToAppName } from '@webapp/models/query';
-import { fetchAppNames } from '@webapp/services/appNames';
+import type { Profile, Groups } from '@pyroscope/models/src';
 import {
   renderSingle,
-  RenderOutput,
   renderDiff,
+  renderExplore,
+  RenderOutput,
+  RenderExploreOutput,
   RenderDiffResponse,
 } from '@webapp/services/render';
-import { Timeline } from '@webapp/models/timeline';
+import { fetchAppNames } from '@webapp/services/appNames';
+import type { AppNames } from '@webapp/models/appNames';
+import { Query, brandQuery, queryToAppName } from '@webapp/models/query';
+import type { Timeline } from '@webapp/models/timeline';
 import * as tagsService from '@webapp/services/tags';
 import { RequestAbortedError } from '@webapp/services/base';
+import { appendLabelToQuery } from '@webapp/util/appendLabelToQuery';
 import type { RootState } from '@webapp/redux/store';
 import { addNotification } from './notifications';
 import { createAsyncThunk } from '../async-thunk';
 
 type SingleView =
   | { type: 'pristine'; profile?: Profile }
   | { type: 'loading'; profile?: Profile }
-  | { type: 'loaded'; timeline: Timeline; profile: Profile }
-  | { type: 'reloading'; timeline: Timeline; profile: Profile };
+  | {
+      type: 'loaded';
+      timeline: Timeline;
+      profile: Profile;
+    }
+  | {
+      type: 'reloading';
+      timeline: Timeline;
+      profile: Profile;
+    };
+
+type ExploreView =
```

Comment:
Why do we need a `timeline` field? From my understanding we only use data from the groups field.

Reply:
removed